### PR TITLE
Changed contact form UI

### LINKED ISSUE #192: Improve Contact Form UI
Required Fields:
- First Name
- Last Name
- Email ID
- Message

Optional Fields:
- Rating ( 1 to 10 slider)
- Phone Number

### PR DESCRIPTION
**Pull request by dipanshparmar**
# changed contact form UI. 
Fixes #192 

## Before
image

## After
image
image

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Doc2pen and 13 more pages - Personal - Microsoft​ Edge 08-10-2020 12_21_49

**Comment by dipanshparmar** [comment]
@anushbhatia I fixed the bug. Please have a look now:
image

**Comment by dipanshparmar** [comment]
@smaranjitghose @anushbhatia

**Comment by smaranjitghose** [comment]
> @anushbhatia I fixed the bug. Please have a look now:
> image

The emojis in the mobile view should be in a single line

**Comment by smaranjitghose** [comment]
contact_html

**Comment by dipanshparmar** [comment]
@smaranjitghose Please have a look now:

image

File: index.html [hunk 1]
```diff
@@ -1,162 +1,250 @@
 <!DOCTYPE html>
-    <html lang="en">
-        <head>
```
Comment (smaranjitghose) [inline index.html:3]: Please do not mess up with the indentation of the parts of the code you are not working on

**Review by smaranjitghose — COMMENTED** [review]
Please ensure that you do not mess up indentation of other parts of the code you are not working on

**Comment by dipanshparmar** [comment]
> Please ensure that you do not mess up indentation of other parts of the code you are not working on

Okay! I'll make sure from the next time :)

**Pull request merged**